### PR TITLE
Initial code.

### PR DESCRIPTION
**Pull request by jvanz**
## Description

Initial code implementing the policy to enforce PSA labels in the namespaces deployed in the cluster.

Fix https://github.com/kubewarden/kubewarden-controller/issues/451

### CODE REVIEW

**Review by flavio — CHANGES_REQUESTED** [review]
LGTM, I left some minor notes

File: src/settings.rs [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+// Describe the settings your policy expects when
+// loaded by the policy server.
+#[derive(Serialize, Deserialize, Default, Debug)]
+#[serde(default)]
+pub(crate) struct Settings {
+    pub modes: Modes,
+}
+
+impl kubewarden::settings::Validatable for Settings {
+    fn validate(&self) -> Result<(), String> {
```
[the remark this answers is not in the document]
Comment (jvanz) [reply src/settings.rs:59]: No reason. It just the current implementation from the [template](https://github.com/kubewarden/rust-policy-template/blob/01173695123c7bbb358f11287c328f996f621317/src/settings.rs#L13)

**Comment by jvanz** [comment]
@jordojordo can you review the questions file that I just added in the PR?

**Comment by jordojordo** [comment]
@jvanz The `questions-ui.yml` looks good! The `artifacthub-pkg.yml` should be good to update with those questions now.

File: Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+crate-type = ["cdylib"]
+
+[dependencies]
+k8s-openapi = { version = "0.18.0", features = ["v1_26"] }
+kubewarden-policy-sdk = "0.9"
+regex="1"
+lazy_static = "1.4"
+serde_json = "1.0"
+serde = {version = "1.0", features = ["derive"] }
+slog = "2.7"
```
Comment (flavio) [inline Cargo.toml:19]: I think this can be removed now

**Review by flavio — CHANGES_REQUESTED** [review]

**Comment by jvanz** [comment]
@flavio I'm merging this PR because I've removed the `slog` dependency.

**Pull request merged**